tests/config/cratedb.ini

```
@@ -1,14 +1,20 @@
 [Datasource]
 Name = cratedb
-LogicalQueryFormat = simple_lower
+LogicalQueryFormat = cratedb
 SchemaName = doc
 CommandLineOverride = FILL_ME_WITH_PATH_OF_REPOSITORY
+MaxThread = 6
 [StandardTests]
-ExpressionExclusions_Standard = math.round.extended
+ExpressionExclusions_Standard = math.round.extended, date.datename.month, date.datename.sow.month, date.dateadd.nulls
```

I don't know much about it: Is it `sow` or `now`?
```suggestion
ExpressionExclusions_Standard = math.round.extended, date.datename.month, date.datename.now.month, date.dateadd.nulls
```

---

It is `sow`, it stands for `Start of week`